charts/zaakbrug/values.yaml

```diff
@@ -97,6 +97,17 @@ affinity: {}
 timeZone: Etc/UTC
 
 zaakbrug:
+  endpoints:
+    soap:
+      beantwoord-vraag:
+        endpoint: "translate/generic/zds/BeantwoordVraag"
+        validation-soft-fail: false
+      ontvang-asynchroon:
+        endpoint: "translate/generic/zds/OntvangAsynchroon"
+        validation-soft-fail: false
+      vrije-berichten:
+        endpoint: "translate/generic/zds/VrijBericht"
+        validation-soft-fail: false
```

Comment:
Is "endpoints".soap.beantwoord-vraag."endpoint" niet dubbel op? T lijkt me wel mooi om die ene laag er tussen uit te halen.

Reply:
Mee eens! Al verwacht ik later dat we zowel soap als rest endpoints zullen hosten, en daarnaast met zowel soap and rest externe API's moeten verbinden. De gedachte was "endpoints" voor ZaakBrug gehoste endpoints en "connectors" voor externe endpoints waar wij mee moeten verbinden.

Voor nu prima om endpoints er tussen uit te halen!

Reply:
Klinkt goed! 

Ik vind het zelf lastig om future proof te zijn met deze helm dingen. We moeten goed in de gaten houden dat als andere mensen upgraden dat ze ook de weizegingen in de values.yaml doorvoeren.